website/i18n/fr/docusaurus-plugin-content-docs/current/providers/provider/completed_todos.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+import 'todo.dart';
+
+/* DÉBUT DU SNIPPET */
```

Comment:
These shouldn't be translated

They are keys used to remove the code before/outside of the comments

Reply:
Ah my bad, I reverted all of them back to English

Notice the build fail (error logs not related to files changed) not quite sure why, and how can I fix it?

Reply:
Ah my bad, I reverted all of them back to English